tests/foreman/cli/test_docker.py

```diff
@@ -117,7 +117,7 @@ def setUpClass(cls):
 
     @tier1
     @run_only_on('sat')
-    def test_create_one_docker_repo(self):
+    def test_positive_create_docker_repo(self):
```

Comment:
As we're inside `DockerRepositoryTestCase` class, no need in `docker_repo` in name. Just `test_positive_create_with_name`